webapi/src/main/scala/org/knora/webapi/responders/v2/SearchResponderV2.scala

```diff
@@ -1159,15 +1939,12 @@ class SearchResponderV2 extends Responder {
                 transformer = triplestoreSpecificQueryPatternTransformerSelect
             )
 
-            // _ = println(triplestoreSpecificPrequery.toSparql)
+            _ = println(triplestoreSpecificPrequery.toSparql)
```

Comment:
Please comment this out, I forgot to do so.

Reply:
also fixed in https://github.com/dhlab-basel/Knora/pull/620/commits/0c13561911483771d29e52f822d66d117c03a4b4